<launch>

<!-- ###### Dependencies ###### --> 
<!-- depends on timed-roslaunch: can be installed by sudo apt install ros-kinetic-timed-roslaunch -->


<!-- #################### hrc_ros package nodes ################################## --> 

<node pkg="hrc_ros" type="observation_agent_IE" name="observation_agent_IE" output="screen" respawn="true" />

<node pkg="hrc_ros" type="task_manager_IE" name="task_manager_IE" output="screen" respawn="true" /> 

<node pkg="hrc_ros" type="robot_motion_agent" name="robot_motion_agent" output="screen" respawn="true" /> 

<node pkg="hrc_ros" type="dobot_worker_agent" name="dobot_worker_agent" output="screen" respawn="true" />

<!-- Start HeadTracking node --> 
<node pkg="hrc_ros" type="HeadTracking_agent" name="HeadTracking_agent" args="-d=4 -l=0.05 --refine=2 --ci=0 --imgshow=0 --printdebug=0"


</launch>
